### ISSUE TITLE
Game Name

### ISSUE THREAD

**Issue by sanderman01** [sanderman01/unity-grids#2]
I want to hear suggestions on you what you think would be a good name for this game.

Bear in mind this game will focus around building both fixed structures and vehicles, and also piloting these vehicles. There will be some forms of resources gathering and crafting towards those ends. I also want to include some engineering concepts, like power generation, and various resource processing machines. Perhaps even stuff like liquids and gases.

Bear in mind the desired properties of a good name for a game:

Must have:

* Not too long and not too short. We do not want naming collisions with common abbreviations. Nor do we want a long name that's hard to remember.
* Not too complicated. If I hear someone say the name out loud for the first time, I should not be wondering how it is spelled.
* Not already in use in the gaming and tech industry.
* No naming collision with another well known concept.
* Exciting.

Want to have:
* Descriptive: The core idea of the game should be readily apparent from the name.
* Evocative/Exciting: The game evokes a feeling that the player will have while playing.

In short, the game name needs to be unique, memorable and easy to search.
